extensions/interactions/MusicNotesInput/directives/MusicNotesInputRulesService.ts

```diff
@@ -43,7 +43,7 @@ oppia.factory('MusicNotesInputRulesService', [
       // TODO(wxy): validate that inputs.a <= inputs.b
       HasLengthInclusivelyBetween: function(answer, inputs) {
         var answerLength = _convertSequenceToMidi(answer).length;
-        return length >= inputs.a && length <= inputs.b;
+        return answerLength >= inputs.a && answerLength <= inputs.b;
```

Comment:
Hi @seanlip, I am not sure if I this change is right. There is no variable length declared in the function and we want to check if answerLength is inclusive between inputs.a and inputs.b (as from the function name and definition). So, is the variable length used here was by mistake?

Reply:
@DubeySandeep can you also please help me here, Thanks!

Reply:
Goodness, yes, I think so. Really good catch. I wonder how it worked before ... :(

Just wondering, are there tests for this currently?

Reply:
Yes, this PR adds test to avoid the use of global variables directly.

Reply:
This was caught since length was flagged as a global variable used directly by eslint. Then, I realized that length here is used by mistake and is not a global variable actually.

Reply:
Ah I meant tests for HasLengthInclusivelyBetween. Think it's worth adding them? They should be fairly trivial.

Reply:
Done.